### ISSUE TITLE
Code responses are getting blocked by the client filters, Oops, your response got filtered 

### ISSUE THREAD

**Issue by Saulul** [microsoft/vscode-copilot-release#127]
Currently the service is completely unusable due to every multi line code response getting cut in the middle and the filtered message being shown. 

- VS Code Version: 
Version: 1.79.0-insider
Commit: 9084e081d4e89ed8ab67fce340d573c4e1378939
Date: 2023-05-15T05:24:19.845Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.4.0
- OS Version:
MacOS Ventura 13.3.1 (a) (22E772610a)
- Logs: N/A

Steps to Reproduce:

1. Ask literally anything that will cause the chat bot to output more than 5 lines of code
2. It will be cut out in the middle and filtered
3. In the example below, I highlighted some sample code and used the /tests command that will write unit tests for the code. The Chat bot successfully started outputting the tests but cut out in the middle and showed this message:
![image](https://github.com/microsoft/vscode-copilot-release/assets/35707165/5ecf80ca-7662-44b6-a4b2-d2b8066b48fe)

**Comment by zhuchangyi** [comment]
I got the same message.

**Comment by hiahiahia0213** [comment]
same issue.

**Comment by russhughes** [comment]
Same issue.

**Comment by augustofretes** [comment]
Yeah, it's getting ridiculous. Microsoft is damaging its own products for pseudo-safety. I'm literally better off using OpenAI's API directly myself than using a paid product. It's not acceptable. It should be better, not worse.

**Comment by Saulul** [comment]
> 

Right on point, a coding assistant shouldn't have "filters"...

**Comment by xa-m** [comment]
exact same problem on "create a chess board" or "i want to create an chess game could you create an chess board for that". the response will be filtered. I didn't understand the problem. Which sentences can trigger the filter system when only thing i'm asking is a chess board.

**Comment by gamousquesORT** [comment]
Same here I asked to explain the name of a Go module 

explain the go Package "sync/atomic"

**Comment by digitarald** [comment]
Duplicate of https://github.com/microsoft/vscode-copilot-release/issues/295
